wm.c

```diff
@@ -1522,6 +1561,11 @@ client_move_to_front(struct client *c)
     if (ws == -1)
         return;
 
+
+    /* If the Client is set to be always below */
+    if (c->below || client_window_is_below(c))
```

Comment:
`client_window_is_below` should probably encapsulate the logic to return the value of `c->below`. In other words, this line should become
```c
if (client_window_is_below(c)) {
```
and the return value for `c->below` would be moved inside that function.